storage/volume/types/volume.go

```diff
@@ -162,9 +155,19 @@ func (v *Volume) SetLabel(label, value string) {
 	v.Labels[label] = value
 }
 
-// Size returns volume's size(MB).
+// Size returns volume's size(bytes).
 func (v *Volume) Size() string {
-	return v.Spec.Size
+	if v.Spec.Size != "" {
```

Comment:
Can we add some verification for the `Size` in API interface?

Reply:
No need 😄